### PR TITLE
bootstrap login and signup

### PR DESCRIPTION
**Pull request by daparikh2500**
Please review and merge to main. If agreed upon,

### CODE REVIEW

**Comment by lbbri** [comment]
Could you be more descriptive with your PR message and maybe add a screenshot of the pages for proof of what it looks like? I think this can be done if you click the `...` in the upper right corner of your message and select edit. Then you can drag and drop or copy and paste the screenshot in.

File: templates/log-in.html [hunk 1]
```diff
@@ -1,5 +1,77 @@
...
+</body>
+
+</html>
+
+
+<!-- <html>
+
 <head>
     <title>Log In</title>
     <style>
```
Comment (lbbri) [inline templates/log-in.html:77]: nit: Could you delete this commented out section - L95.

**Review by lbbri — APPROVED** [review]
I don't know much about styling, and the pages looked awesome! Besides the one nit, LGTM!

File: templates/log-in.html [hunk 2]
```diff
@@ -1,5 +1,77 @@
...
+                            <br>
+
+                            <div class="form-group">
+                                <i class="fas fa-lock"></i>
+                                <input class="myInput" type="password" placeholder="Password" id="password" required>
+                            </div>
+
+                            <input type="submit" class="butt" value="LOG IN"><BR></BR>
+                            <p>OR</p>
+                            <input type="submit" class="butt" value="CREATE AN ACCOUNT">
```
Comment (khoacao1) [inline templates/log-in.html:38]: This should be the same format as:
`<p>Already have an account? <a href="{{ url_for('login') }}">Log In</a></p>`
and stay outside of the form.

**Pull request merged**